crc-e2e/task.yaml

```
@@ -0,0 +1,325 @@
+      description: set the name for the junit results file for integration
+      default: integration-junit.xml
+    - name: integration-cleanup-target
+      description: To cleanup remote folder after running integration
+      default: "true"  
+    - name: worspace-qe-subpath
+      description: subpath relative to workspace path where results are stored
+      default: qe-results
+    - name: debug
+      description: debug purposes extend verbosity on cmds executed on the target
+      default: 'false'
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)